packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -51,7 +51,7 @@ export const SideNavMethods: FC<MethodsProps> = ({ methods, tag, specKey }) => {
             <SidebarItem key={method.name} as="span">
               <>
                 <MethodBadge verb={method.httpMethod} />
-                {highlightHTML(pattern, method.operationId)}
+                {highlightHTML(pattern, method.summary)}
```

Comment:
Are we going to do a truncate here, or are you waiting until switching to `Accordion`?

Reply:
I'll get that in here. No reason to wait.